Formula/l/llama.cpp.rb

```diff
@@ -11,6 +11,7 @@ class LlamaCpp < Formula
   livecheck do
     url :stable
     regex(/^v?b(\d+(?:\.\d+)*)$/i)
+    throttle 5
```

Comment:
we tried it in 

- #173185
- https://github.com/Homebrew/homebrew-core/pull/173213